File: fieldsync-inventory-backend/src/main/java/com/example/fieldsync_inventory_api/dto/user/role/RoleResponseDTO.java
package com.example.fieldsync_inventory_api.dto.user.role;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.Instant;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class RoleResponseDTO {
    private Integer id;
    private String name;
    private String description;
    private Instant createdAt;
    private Instant updatedAt;
    private Instant deletedAt;
}

File: fieldsync-inventory-app/app/src/main/java/com/example/fieldsync_inventory_app/data/remote/dto/role/RoleResponseDto.kt
package com.example.fieldsync_inventory_app.data.remote.dto.role

data class RoleResponseDto(
    val id: Int,
    val name: String,
    val description: String?,
    val createdAt: String,
    val updatedAt: String,
    val deletedAt: String?
)
